### PR TITLE
feat: Added neon theme

### PR DESCRIPTION
**Pull request by anirban12d**
This theme has bright colors that works well with dark backgrounds.

## Description

Fixes # 

### Type of change

- [ ] Bug fix (added a non-breaking change which fixes an issue)
- [x] New feature (added a non-breaking change which adds functionality)
- [ ] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] Tested locally with a valid username
- [ ] Tested locally with an invalid username
- [ ] Ran tests with `composer test`
- [ ] Added or updated test cases to test new features

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/themes.php [hunk 1]
```diff
@@ -1358,4 +1358,16 @@
         "sideLabels" => "#8D192B",
         "dates" => "#862931",
     ],
+     "neon" => [
+        "background" => "#000000",
+        "border" => "#ffffff",
+        "stroke" => "#ffffff",
```
Comment (DenverCoder1) [inline src/themes.php:1364]: Minor thing for consistency

```suggestion
        "border" => "#FFFFFF",
        "stroke" => "#FFFFFF",
```

**Review by DenverCoder1 — COMMENTED** [review]
Looks great :+1: 

Would you be able to also update the docs - <https://github.com/anirban12d/github-readme-streak-stats/blob/patch-1/docs/themes.md>?

image

Here's an image url that can be used - https://user-images.githubusercontent.com/20955511/225303106-8c901c48-732e-49ae-a2e6-8733254536eb.png

Let me know if you have any questions.

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Comment by anirban12d** [comment]
You are most welcome. 🎉

**Pull request merged**